src/components/App/App.js

```diff
@@ -5,11 +5,13 @@ import Header from '../Header/Header';
 import PrivateRoute from '../Utils/PrivateRoute';
 import PrivateRouteShelter from '../Utils/PrivateRouteShelter';
 import PublicRoute from '../Utils/PublicRoute';
+import PublicRouteShelter from '../Utils/PublicRouteShelter';
 import AuthContext from '../../context/AuthContext';
 import AddPetPage from '../../routes/AddPetPage/AddPetPage';
 import FavoritesPage from '../../routes/FavoritesPage/FavoritesPage';
 import HomePage from '../../routes/HomePage/HomePage';
 import LoginPage from '../../routes/LoginPage/LoginPage';
+import ShelterLoginPage from '../../routes/ShelterLoginPage/ShelterLoginPage';
```

Comment:
Do you think you can use `../../routes/LoginPage/LoginPage` for shelter login? Both `LoginPage` and `ShelterLoginPage` are pretty much the same except `ShelterLoginPage` has `setShelterAdminState(true);` and the html txt `Shelter Login`.
I think we can pass `props` from the parents (in this case, it's `App.js`) see https://reactjs.org/docs/components-and-props.html#rendering-a-component and https://github.com/asching7108/purrfect-match/pull/7/files#diff-564721b31db15d46e221a91e223826cdc7bd4f526bddd2b5d40100fa7469b458 line 72 `<PetCard className={cName} key={pet.PetID} pet={pet} />`. If it's too complicated than what I suggest, you don't have to do it. Just suggection!

Reply:
I am guessing that you can do something like `<LoginPage userType="shelter"/>`. If userType is null/undefined, set the page for a user, if userType is shelter, set the page for shelter.

Reply:
Good note! Just sent a fix, thanks!